client/components/plans/plan-actions/style.scss

```diff
@@ -171,3 +171,7 @@
 .jetpack_business.plan-actions__illustration {
 	background-image: url('/calypso/images/plans/plan-business.svg');
 }
+
+.jetpack_premium .is-image-button, .jetpack_business .is-image-button {
+	display:none;
```

Comment:
Do we need to do this in CSS? If we do, you should have a space between the `:` and the `none`.